scipy-1.0/paper.tex

```diff
@@ -974,16 +974,17 @@ \subsubsection*{Benchmark suite}
 The airspeed velocity (asv\cite{asvref}) library enables benchmarking Python packages
 over their lifetimes, and the performance of the SciPy code base was monitored
 with asv starting in February of 2015 (PR \#4501). In addition to ensuring that
-unit tests are passing (see above), confirming that performance generally
+unit tests are passing (see below), confirming that performance generally
 remains constant or improves over the commit hash history of the project allows
 us to objectively measure that our code base is improving, to empower
 scientific applications.
 
 Consider the asv benchmark results shown in Figure~\ref{fig:asvbench}, spanning
 roughly nine years of project history. These demonstrate the gradual
 performance improvements in a nearest-neighbor search through
-\texttt{scipy.spatial.cKDTree.query()}, and can be run using the command:
-
+\texttt{scipy.spatial.cKDTree.query()}, and can be run using the
+following command, where \texttt{run.py} is a light wrapper
+script around \texttt{asv}, packaged with SciPy:
 
 \texttt{python run.py run -e -s 800 --bench "\textbackslash
```

Comment:
I'd suggest a comma at the end of this line, and _then_ "where \texttt{run.py} is a light wrapper
script around \texttt{asv} that is packaged with SciPy."

Reply:
done